controllers/gateway_controller.go

```diff
@@ -188,59 +197,68 @@ func (r *GatewayReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			if err = r.patchStatus(ctx, gateway, oldGateway); err != nil {
 				return ctrl.Result{}, err
 			}
-			return ctrl.Result{}, nil
+			debug(log, "gateway is ready", gateway)
 		}
 	}
 
-	debug(log, "gateway reconciled", gateway)
+	debug(log, "reconciliation complete for Gateway resource", gateway)
 	return ctrl.Result{}, nil
 }
 
-func (r *GatewayReconciler) provisionDataPlane(ctx context.Context, log logr.Logger, gateway *gatewayDecorator, gatewayConfig *operatorv1alpha1.GatewayConfiguration) *operatorv1alpha1.DataPlane {
+func (r *GatewayReconciler) provisionDataPlane(ctx context.Context,
+	log logr.Logger,
+	gateway *gatewayDecorator,
+	gatewayConfig *operatorv1alpha1.GatewayConfiguration) *operatorv1alpha1.DataPlane {
```

Comment:
```suggestion
func (r *GatewayReconciler) provisionDataPlane(
	ctx context.Context,
	log logr.Logger,
	gateway *gatewayDecorator,
	gatewayConfig *operatorv1alpha1.GatewayConfiguration,
	) *operatorv1alpha1.DataPlane {
```

nit. Was that manually formatted or via a tool? `gofumpt`?

Reply:
manually formatted.